src/loaders/MaterialLoader.js

```diff
@@ -6,7 +6,26 @@ import { Matrix3 } from '../math/Matrix3.js';
 import { Matrix4 } from '../math/Matrix4.js';
 import { FileLoader } from './FileLoader.js';
 import { Loader } from './Loader.js';
-import { Material } from '../materials/Material.js';
+import {
```

Comment:
Maybe use just `import * from`?

Reply:
I need the explicit names in the object, it's better like this